### ISSUE TITLE
release the Python gfx module in PyPI

### ISSUE THREAD

**Issue by petri** [matthiaskramm/swftools#7]
It seems to me the Python gfx library provided as part of swftools is very useful. It would be great if that could somehow be packaged separately and released at PyPI (Python Package Index) so it would be easy to find and use.

**Comment by retsyo** [comment]
I agree, especially gfx for python 2/3 on windows. I found it is hard for me to compile in msys2+mingw32, and python 3 is not supported.

**Comment by retsyo** [comment]
Once I have used gfx for py2.4(perhaps, I forgot) on Windows to convert PDF to images. However, I can't use it again on anaconda my python2.7(32 bits) and python 3.4(64 bits)
